backend/app/usecase/url/urlcreator_test.go

```diff
@@ -93,13 +94,8 @@ func TestURLCreatorPersist_CreateURL(t *testing.T) {
 			},
 		},
 		{
-			name: "automatically generate alias",
-			urls: urlMap{
-				"220uFicCJj": entity.URL{
```

Comment:
@byliuyang I wasn't sure why this URL was in the fake in-memory URL map even though the alias wasn't being used in the test case so I took it out for now, but can add it back in if it's actually needed.